services/coingecko/priceService.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from "react"
+import { manifest } from "@/manifests"
+import axios from "axios"
+
+import globalConfig from "@/config/globalConfig"
+
+const coinId = globalConfig.ordersDisplayCurrency.symbol.toLowerCase()
```

Comment:
Are you sure this is always the case?

Reply:
I think we need to have a specific setting for this and then either:
- Guide the user to find the coin id
- Add the coin id for all our supported network in our network list file

Reply:
> Are you sure this is always the case?

yes, always in lowercase